tests/test_async_sqla.py

```diff
@@ -1,62 +0,0 @@
-from unittest.mock import AsyncMock, patch
-
-from pytest import fixture, mark, raises
-from sqlalchemy import text
-
-pytestmark = [mark.sqlalchemy("1.4"), mark.require_asyncpg]
-
-
-@fixture
-async def startup(environ):
```

Comment:
The tests related to startup were moved to `test_startup` and the ones related to `open_session` moved to `test_open_session`. Consider moving this to its own PR.